pkg/btree/bptree.go

```diff
@@ -3,31 +3,31 @@ package btree
 import (
 	"bytes"
 	"fmt"
+	"github.com/kevmo314/appendable/pkg/common"
```

Comment:
`common` isn't a great name for a package, by definition everything is common if it's in a package. :)

Maybe `ptr` or something?